src/style.css

```diff
@@ -1,3 +1,128 @@
-body {
-    background-color: bisque;
-}
\ No newline at end of file
+.cover-page {
+  background-image: url('./assets/the-witcher-season-2.jpg');
+  height: 100vh;
+}
+
+h1 {
+  font-size: 48px;
+}
+
+.comments-form-popup {
+  display: none;
+  position: fixed;
+  bottom: 25%;
+  right: 35%;
+  left: 35%;
+  z-index: 9;
+  border-radius: 4px 3%;
+}
+
+.form-container {
+  max-width: 100%;
+  padding: 10px;
+  background-color: white;
+}
+
+.form-container input[type=text] {
+  width: 100%;
+  padding: 10px;
+  margin: 5px 0 22px 0;
+  background: transparent;
+}
+
+.form-container input[type=text]:focus {
+  background-color: #f1f1f1;
+  outline: none;
+}
+
+.form-container .btn {
+  background-color: black;
+  color: white;
+  padding: 10px 15px;
+  border: none;
+  cursor: pointer;
+  width: 100%;
+  margin-bottom: 10px;
+  opacity: 0.8;
+}
+
+.form-container .cancel {
+  background-color: blue;
+}
+
+.form-container .btn:hover,
+.open-button:hover {
+  opacity: 1;
+}
+
+.likes {
+  cursor: pointer;
+}
+
+#overlay {
+  display: none;
+  position: absolute;
+  top: 0;
+  bottom: 0;
+  background: #999;
+  width: 100%;
+  height: 100%;
+  opacity: 0.8;
+  z-index: 100;
+}
+
+#popup {
+  /*
+  position: fixed;
+  top: 50%;
+  left: 50%;
+  width: 200px;
+  margin-left: -100px;
+  margin-top: -100px;
+  z-index: 200; */
```

Comment:
-  [OPTIONAL] Kindly remove commented codes to have a cleaner and professional code.